guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{Project} can manage DHCP service using your {SmartProxy}.
```

Comment:
I'd say `integrate with` is better. It can manage ISC DHCP but all others only talk using APIs.
```suggestion
{Project} can integrate with a DHCP service using your {SmartProxy}.
```

Reply:
Applied.